deploy/docker/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM progrium/busybox
+FROM alpine:3.2
 MAINTAINER vishnuk@google.com
 
-# cAdvisor discovery via external files.
-VOLUME /var/run/heapster/hosts
-
-RUN opkg-install ca-certificates
+RUN apk --update add curl ca-certificates tar && \
+    curl -Ls https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk > /tmp/glibc-2.21-r2.apk && \
```

Comment:
I guess you haven't tried building this? That URL doesn't work IIRC. Try changing the 6 to 8. Relying on build artifacts isn't great but no other option right now if using alpine.